src/message/header/content_type.rs

```diff
@@ -67,6 +67,14 @@ impl FromStr for ContentType {
     }
 }
 
+#[cfg(feature = "mime03")]
+#[cfg_attr(docsrs, doc(cfg(feature = "builder")))]
```

Comment:
This needs to say `mime03`. Sorry I left the review from my phone 😅

Reply:
Oh duh, sorry for blindly copying and pasting lol